### ISSUE TITLE
foldingStartMarker and foldingStopMarker ignored?

### ISSUE THREAD

**Issue by spencerlyon2** [atom/first-mate#48]
When I try to fold text in atom it seems that it is indentation based and these two parts of the TextMate preferences (settings) are ignored.

Are they currently active in atom? If so, where is a good example of their usage so I can try to fix the problem with my grammar?

If not, I suppose this is a request that the feature be added :smile:

**Comment by spencerlyon2** [comment]
Also, I should note that I do see I can fold selections. This is great, but is not the same as the start and end textmate markers.

**Comment by izuzak** [comment]
@spencerlyon2 Yeah, folding in Atom is currently indentation based. Added the "enhancement" label to this issue so that the team can consider the request :+1:.

**Comment by spencerlyon2** [comment]
Great thanks. Hopefully I found the right repo to post the issue under

**Comment by jugglingnutcase** [comment]
This would be a helpful addition for languages like C# and powershell, which use regions that often don't have indented code underneath them. ([C# example](https://github.com/NancyFx/Nancy/blob/188bc90450b10d263495d0a33c4cec98a1903937/src/Nancy.ViewEngines.Razor/RazorConfigurationSection.cs#L111-122))

**Comment by Hubro** [comment]
Is this being actively worked on? Is it on a roadmap somewhere? Being able to customize code folding is a pretty glaring omission from "The Hackable Editor" :P

**Comment by ThatXliner** [comment]
What's the status on this
